_connect-content/guides/api/select-tables-and-fields.md

```diff
@@ -0,0 +1,309 @@
+---
+# -------------------------- #
+#          PAGE INFO         #
+# -------------------------- #
+
+title: Select streams and fields with the Connect API
+content-type: "tutorial"
+content-id: "select-tables-and-fields"
+layout: tutorial
+
+permalink: /stitch-connect/guides/select-tables-and-fields-with-connect-api
+
+toc: false
+summary: false
+feedback: false
+
+summary: "Using the Stitch Connect API, select a source's streams and fields for replication using this step-by-step tutorial."
+
+
+# -------------------------- #
+#   RELATED SIDEBAR LINKS    #
+# -------------------------- #
+
+related:
+  - title: "Connect API reference"
+    link: "{{ link.connect.api | prepend: site.baseurl }}"
+
+  - title: "Create and configure a source with Connect"
+    link: "{{ link.connect.guides.create-configure-a-source | prepend: site.baseurl }}"
+
+  - title: "Field selection and compatibility rules"
+    link: "{{ link.connect.guides.field-selection-compatibility-rules | prepend: site.baseurl }}"
+
+  - title: "Connect guides"
+    link: "{{ link.connect.guides.category | prepend: site.baseurl }}"
+
+
+# -------------------------- #
+#         GUIDE INTRO        #
+# -------------------------- #
+
+intro: |
+  {% include misc/data-files.html %}
+
+  {{ page.summary }}
+
+
+# -------------------------- #
+#         GUIDE INTRO        #
+# -------------------------- #
+
+requirements:
+  - item: |
+      **Valid credentials**. Using the Connect API requires a valid [access token]({{ site.data.connect.api.section | flatify | prepend: site.baseurl | append: site.data.connect.api.access-api }}), which is obtained through partner credentials.
+  - item: |
+      **Access to a source**. This guide will use a [Shopify SaaS source]({{ site.data.connect.api.section | flatify | prepend: site.baseurl | append: site.data.connect.data-structures.source-form-properties.section |  append: "-shopify-object" }}) as an example, but any source type will work. Refer to the [Source Form Properties documentation]({{ site.data.connect.api.section | flatify | prepend: site.baseurl | append: site.data.connect.data-structures.source-form-properties.section }}) for the sources supported by Connect.
+
```

Comment:
This actually only works on a source with a field_selection step. Might want to include that info here.

Reply:
Good point! I'll add that in.